examples/undocumented/python/graphical/classifier_gaussian_process_binary_classification.py

```diff
@@ -73,35 +64,35 @@ def gaussian_process_binary_classification_laplace(X_train, y_train, n_test=50):
     # inf = SingleLaplacianInferenceMethod(kernel, train_features, mean, train_labels, lik)
 
     # specify EP approximation inference method
-    inf = EPInferenceMethod(kernel, train_features, mean, train_labels, lik)
+    inf = sg.EPInferenceMethod(kernel, train_features, mean, train_labels, lik)
 
     # create and train GP classifier, which uses Laplace approximation
-    gp = GaussianProcessClassification(inf)
+    gp = sg.GaussianProcessClassification(inf)
```

Comment:
all those should have a factory, meta examples should guide you

Reply:
just checked, it is not possible currently, and we need to refactor this stuff (meta example is also not yet ported to new api I realised). Ignore my comment then

Reply:
#4996